tools/parse/src/main/scala/org/allenai/nlpstack/parse/poly/fsm/FSMTrainingVectorSource.scala

```diff
@@ -34,7 +36,7 @@ abstract class FSMTrainingVectorSource(
     if (!vectorIter.hasNext) {
       resultSoFar
     } else {
-      taskHelper(resultSoFar + vectorIter.next.task, vectorIter)
+      taskHelper(resultSoFar + vectorIter.next().task, vectorIter)
     }
```

Comment:
My assumption is that getVectorIterator.map(_.task) would create some kind of in-memory sequence of tasks before converting it to a set (via the .toSet command).